### ISSUE TITLE
Doubts about masks

### ISSUE THREAD

**Issue by xgiaogiao** [xintangjin/DMDC#1]
Hello, your results are awesome, is the mask in ARAD_1K the mask in the TSA_simu_data

**Comment by xintangjin** [comment]
Yes, I am using the the mask in the TSA_simu_data for my training. But I have a feeling a randomized Mask should work as well.

**Comment by xgiaogiao** [comment]
Thank you for your reply!
